### ISSUE TITLE
[Bug] Cannot RUN when PULLing Amazonliunux Image in Docker Image

### ISSUE THREAD

**Issue by topias11a705** [amazonlinux/amazon-linux-2023#343]
https://hub.docker.com/_/amazonlinux

Use This image on ARM64bit 
https://hub.docker.com/layers/library/amazonlinux/2023.0.20230503.0/images/sha256-97cab6a5e0a46c03f66715e8a3ac7ef12f5d727b8d9bd246da5ad607570a88e5?context=explore
```
root@ubuntu:/home/xxxxx# docker images -a
REPOSITORY                TAG                IMAGE ID       CREATED         SIZE
amazonlinux               <none>             **a437c001bf61**   3 days ago      178MB

root@ubuntu:/home/xxxxx# docker run -it **a437c001bf61**
root@ubuntu:/home/xxxxx# docker ps -a
CONTAINER ID   IMAGE                   COMMAND       CREATED              STATUS                        PORTS     NAMES
aa744bfe3657   a437c001bf61            "/bin/bash"   About a minute ago   **Exited (132) 34 seconds ago**             mystifying_euler
``` 

I have PULLed and RUN the image for AMD Windows 11 but still get the same error(Exited (132)) .
I'm not sure if this is a build issue or an AmazonLinux issue so I'm reporting it here.

Docker version used on Arm64bit computer is **23.0.5, build bc4487a**

![image](https://user-images.githubusercontent.com/38270203/236702286-e650b550-09b5-482c-8525-c153a01b2708.png)

**Comment by ozbenh** [comment]
What specific ARM64 computer is this ? Amazon Linux 2023 is built with `-march=armv8.2-a+crypto` which means it needs a fairly modern ARM64 core with at least ARMv8.2 support with the optional Cyrptographic Extension. Afaik this includes Cortex A75/A55 and later (and N1, V1 etc...)

**Comment by rodmaz** [comment]
@ozbenh  That probably answers the question why we cannot run Amazon Linux 2023 on our Raspberry Pi 3 (Broadcom BCM2837).

**Comment by ozbenh** [comment]
Right. We are looking at printing a more informative error message in a future update. Unfortunately no RPi has a new enough ARM core (though some Rockship RK3588 based devices *might* work, I haven't tried)
